gobblin-core/src/main/java/org/apache/gobblin/writer/FsDataWriter.java

```diff
@@ -96,7 +97,7 @@ public FsDataWriter(FsDataWriterBuilder<?, ?> builder, State properties) throws
     this.writerAttemptIdOptional = Optional.fromNullable(builder.getWriterAttemptId());
     this.encoders = builder.getEncoders();
 
-    Configuration conf = new Configuration();
+    this.conf = new Configuration();
```

Comment:
We can populate the conf object with the properties passed in the constructor so that users can pass on custom configs like for Trash / File System-
`this.conf = HadoopUtils.getConfFromProperties(properties);`

Reply:
Line 102 might be doing the same thing.